cylc/flow/scripts/install.py

```diff
@@ -285,6 +285,7 @@ def install_cli(
 def install(
     opts: 'Values', reg: Optional[str] = None
 ) -> Tuple[str, str]:
+    set_timestamps(LOG, opts.verbosity > 1)
```

Comment:
I think that you need this:

```suggestion
    set_timestamps(LOG, opts.verbosity > 1 and opts.log_timestamp)
```

If you don't I think that

```bash
cylc install --debug --no-timestamp
```

Gets a timestamp.

Reply:
(would need to test this with `vip`)